content/docs/integrations/kubernetes/index.md

```diff
@@ -10,423 +10,177 @@ integrationType: OSS plugin
-
-2. Search for IAM in the services box and then click on ”Roles” on the left handside tab.
-
-3. Search for your newly created role: ”**[your company name]**-roadie-read-only-role” and click on it.
-You should see a page like below. Record the role arn. This is your assumable role arn which you'll need to configure RBAC in kubernetes.
-
-![role-page](./role-page.png)
-
-3. Click on ”Trust Relationships”, then ”Edit relationship” and add the text below:
-
-``` json
-{
-  "Version": "2012-10-17",
-  "Statement": [
-    {
-      "Effect": "Allow",
-      "Principal": {
-        "AWS": [
-          "ACCOUNT ARN SUPPLIED ON THE CONFIGURATION PAGE"
-        ]
-      },
-      "Action": "sts:AssumeRole",
-      "Condition": {
-        "StringEquals": {
-          "sts:ExternalId": "EXTERNAL ID AS CONFIGURED ABOVE"
-        },
-        "StringLike": {
-          "aws:PrincipalArn": [
-            "*[Roadie backend role name]"
-          ]
-        }
-      }
-    }
-  ]
-}
+### Step 2. Create the Cluster Role and binding.
+1. Connect to your EKS cluster [in your terminal](https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html). 
+2. [Apply](https://helm.sh/docs/intro/install/) the Helm Chart to your cluster:
+```shell
+helm repo add roadie https://charts.roadie.io
+helm install roadie-kubernetes-cluster-access roadie/roadie-kubernetes-cluster-access \
+# Only set the following if using the Broker:
+#   --set broker.enabled=true \
+#   --set broker.token=<some-random-key> \
+#   --set broker.tenantName=<your-roadie-tenant-name>
 ```
- > In the Json fragment above, replace the "[Roadie backend role name]" with the role that is provided to you on the Kubernetes configuration page (See step 1). Note the leading `*` is required.
 
-4. Save the changes.
+### Step 3. Add the Role Mapping in aws-auth
+1. Log into your cluster
+2. Edit your Kubernetes `aws-auth` pod's Configmap as per [the EKS docs](https://docs.aws.amazon.com/eks/latest/userguide/add-user-role.html).
+i.e. manually `kubectl edit -n kube-system configmap/aws-auth` or using eksctl `eksctl create iamidentitymapping --cluster <my-cluster> --region=<region-code> \
+--arn arn:aws:iam::<your-account-number>:role/roadie-roadie-read-only-role --username roadie --group system:authenticated`
 
-### Step 3: Set RBAC for new role
-
-1. Edit your Kubernetes aws-auth Configmap as per [the EKS docs](https://docs.aws.amazon.com/eks/latest/userguide/add-user-role.html).
-
-It should look something like this:
+ℹ️ NB: The role arn is the same as the arn you copied from the Output `RoadieIamRoleArn` after running the Cloudformation template in step 1. 
 
+Running `kubectl describe -n kube-system configmap/aws-auth` the new entry should look something like this:
 ``` yaml
  - mapRoles:
    - "groups":
       - "system:authenticated"
-      "rolearn": "[ASSUMABLE ROLE ARN]"
+      "rolearn": "arn:aws:iam::<your-account-number>:role/roadie-roadie-read-only-role"
       "username": "roadie"
```

Comment:
I dont think this matches the name from the helm chart? or maybe it has changed since I last looked.